app/models/user.rb

```diff
@@ -8,15 +8,14 @@ class User < ActiveRecord::Base
 
   accepts_nested_attributes_for :authentications
 
-  validates :password, length: { minimum: 6, message: "Длина пароля должна быть не меньше 6 символов" },
-            if: -> { new_record? || changes["password"] }
-  validates :password, confirmation: { message: "Пароли не совпадают" },
-            if: -> { new_record? || changes["password"] }
-  validates :password_confirmation, presence: { message: "Поле не может быть пустым" },
-            if: -> { new_record? || changes["password"] }
+  validates :email, uniqueness: { message: 'Такой email уже существует' },
+            presence: { message: "Поле не может быть пустым" },
+            :format => { with: /\A[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]+\z/,
```

Comment:
Line is too long. [81/80]<br>Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.